packages/server/package.json

```diff
@@ -44,6 +44,9 @@
     "url": "https://github.com/open-feature/js-sdk/issues"
   },
   "homepage": "https://github.com/open-feature/js-sdk#readme",
+  "engines": {
+    "node": ">=16"
+  },
```

Comment:
Just a note for others. Previously the server SDK was requiring node 14, but that's very dead. We are upgrading to 16 here but not considering it a breaking change since the version is so old (even 16 is partially EOL).

https://endoflife.date/nodejs